source/frontend/src/features/research/list/ResearchFilter/AppCreateUpdateRangeSelect.tsx

```diff
@@ -98,6 +98,6 @@ const SmallInputGroup = styled(InputGroup)`
 `;
 
 const SmallDatePicker = styled(FastDatePicker)`
-  width: 18rem;
+  min-width: 12rem;
```

Comment:
This allows a bit of scaling for different screen sizes, but ensures a minimal display with readable date text.